learn/what_is_meilisearch/search_preview.md

```diff
@@ -0,0 +1,23 @@
+# Search preview
+
+After adding documents to your Meilisearch instance, you can immediately start searching through your dataset using Meilisearch's search preview. This can be useful when you want to check documents have been properly indexed and Meilisearch is working as expected.
```

Comment:
```suggestion
After adding documents to your Meilisearch instance, you can immediately start searching through your dataset using Meilisearch's search preview. This can be useful when you want to check that documents have been properly indexed and Meilisearch is working as expected.
```